cypress/e2e/dashboard/events/index.cy.js

```diff
@@ -267,9 +274,13 @@ describe('events dashboard', () => {
     cy.get('@eventToEdit').invoke('attr', 'href').as('eventHref');
 
     cy.findByRole('link', { name: 'Dashboard' }).click();
+
     cy.findByRole('link', { name: 'Events' }).click();
+    cy.wait('@gqleventsQuery');
```

Comment:
```suggestion
    cy.wait('@GQLevents');
```
If we use the above command, this needs changing.